### PR TITLE
Add instruction to restart dbus for Raspberry Pi OS

### PR DESCRIPTION
**Pull request by c-goes**
See also: #358

### CODE REVIEW

**Review by lsm5 — APPROVED** [review]
LGTM

**Comment by lsm5** [comment]
@TomSweeneyRedHat @rhatdan @DanBmh PTAL

File: getting-started/installation.md [hunk 1]
```diff
@@ -204,13 +204,17 @@ echo 'deb http://deb.debian.org/debian buster-backports main' | sudo tee -a /etc
...
-# Add missing keys manually
+# Add missing keys for buster-backports manually
 sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 04EE7237B7D453EC
 sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 648ACFD622F3D138
 
 sudo apt-get update
 sudo apt-get -y -t buster-backports install libseccomp2
 sudo apt-get -y install podman
+# Restart dbus for rootless podman. Do this for every user using containers.
+# This command only works without root. One way to do this is login as the
```
Comment (TomSweeneyRedHat) [inline getting-started/installation.md]:
```suggestion
# This command only works without root. One way to do this is to login as the
```

**Comment by TomSweeneyRedHat** [comment]
One very small nit, but as it's user-facing documentation, I'd appreciate it if you could touch it up.  I'll merge it in once complete.

**Comment by DanBmh** [comment]
Its now working for me. Not sure what did change on the pi or my login, I've seen that you didn't change the command.

**Comment by lsm5** [comment]
> Its now working for me. Not sure what did change on the pi or my login, I've seen that you didn't change the command.

wasn't the issue earlier that you were running `systemctl --user` as root?

**Comment by DanBmh** [comment]
Yes, but as far as I remember I did try it without root too, and I was asked for some password, which often happens if I try to run some root only commands without sudo. So I thought I need sudo for this.

**Comment by lsm5** [comment]
> Yes, but as far as I remember I did try it without root too, and I was asked for some password, which often happens if I try to run some root only commands without sudo. So I thought I need sudo for this.

Ack, that sounds strange, but let us know if you see it again.

**Review by lsm5 — APPROVED** [review]
LGTM.

**Pull request merged**